website/registrations/utils.py

```diff
@@ -336,12 +360,12 @@ def _validate(self):
                     except AttributeError:
                         self.log_error(invalid=True, type='invalidContributors')
                     else:
-                        parsed_value.append({'full_name': full_name, 'email': email})
+                        parsed_value.append({'full_name': full_name.strip(), 'email': email.strip()})
             self._parsed_value = parsed_value
 
 class LicenseField(MetadataField):
     # format: license_name;year;copyright_holder_one,copyright_holder_two,...
-    with_required_fields_regex = re.compile(r'(?P<name>[\w ]+);(?P<year>[ ][1-3][0-9]{3});(?P<copyright_holders>[\w -,]+)')
+    with_required_fields_regex = re.compile(r'(?P<name>[\w ]+);(?P<year>[ ][1-3][0-9]{3});(?P<copyright_holders>[\w -,/]+)')
     no_required_fields_regex = re.compile(r'(?P<name>[\w ]+)')
```

Comment:
We might need to tweak this regex later. There are license (with no required fields that) whose name contains non `\w` characters. Screenshots are from my local env but I think they are the same on servers.

<img width="881" alt="Screen Shot 2021-09-24 at 10 03 18 AM" src="https://user-images.githubusercontent.com/3750414/134687607-5e170075-1fa0-466f-80e4-4f09cc621b43.png">

Reply:
Fixed